src/graphics/Window.cpp

```diff
@@ -0,0 +1,79 @@
+//
+// Created by ajer on 11.02.19.
+//
+
+#include "Window.h"
+#include <glad/glad.h>
+#include <GLFW/glfw3.h>
+
+namespace Bald::Graphics {
+
+
+    Window::Window(const char *title, int width, int height, bool closed, bool VSync)
+            : m_Title(title), m_Width(width), m_Height(height), m_Closed(closed), m_VSync(VSync) {
+        Window::Init();
+    }
+
+    void Window::Update() noexcept {
```

Comment:
Update should ONLY PollEvents and SwapBuffers!!!! Viewport should be changed once per FrameBufferCallback.

Reply:
done